smac/facade/hyperparameter_optimization_facade.py

```diff
@@ -178,7 +178,7 @@ def get_random_design(  # type: ignore
         probability : float, defaults to 0.2
             Probability that a configuration will be drawn at random.
         """
-        return ProbabilityRandomDesign(probability=probability)
+        return ProbabilityRandomDesign(probability=probability, seed=scenario.seed)
```

Comment:
I agree that this needs to be correctly passed to `ProbabilityRandomDesign`. However, since the default parameter for  `seed` is set as 0, ignoring this part might not result in any non-determinism behaviors. Could you please provide more details regarding the non-determinism things?

Reply:
https://github.com/automl/SMAC3/pull/1066#issuecomment-1749127210